src/helpers/remind-and-remove.ts

```diff
@@ -96,9 +96,12 @@ async function removeAllAssignees(context: ContextPlugin, issue: ListIssueForRep
     return false;
   }
   const logins = issue.assignees.map((o) => o?.login).filter((o) => !!o) as string[];
-  const logMessage = logger.info(`Passed the deadline and no activity is detected, removing assignees: ${logins.map((o) => `@${o}`).join(", ")}.`, {
-    issue: issue.html_url,
-  });
+  const logMessage = logger.info(
+    `Passed the disqualification threshold and no activity is detected, removing assignees: ${logins.map((o) => `@${o}`).join(", ")}.`,
```

Comment:
I wonder if its redundant to tag the assignees and then remove them. We can leave it for now but just a thought to consider.

Reply:
From what I've seen you get no notification when un-assigned, so I think this might be useful?